.github/PULL_REQUEST_TEMPLATE/standard-change.md

```diff
@@ -8,9 +8,9 @@
 
  <-- Please fill the boxes with [x] before submitting a pull request --> 
 
-- [ ] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
-- [ ] I have run `yarn build` after adding my changes **without getting any errors**. 
+- [] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
+- [] I have run `yarn build` after adding my changes **without getting any errors**. 
 
 ## Updating documentation or Bugfix
 
-<-- Help us understand your motivation by explaining why you decided to make this change. We must be able to understand the purpose of your change from this description.  Does this fix a bug? Does it close an issue? If so please mention it.  -->
\ No newline at end of file
+<-- Help us understand your motivation by explaining why you decided to make this change. We must be able to understand the purpose of your change from this description.  Does this fix a bug? Does it close an issue? If so please mention it.  -->
```

Comment:
Looks like the last newline at the end of this file was either added or removed: difficult to fix from the GitHub editor so I think you will have to submit this change yourself from your local git.

In any case before we merge this PR there should be no changes to the file `.github/PULL_REQUEST_TEMPLATE/standard-change.md`.

Reply:
> Looks like the last newline at the end of this file was either added or removed: difficult to fix from the GitHub editor so I think you will have to submit this change yourself from your local git.
> 
> In any case before we merge this PR there should be no changes to the file `.github/PULL_REQUEST_TEMPLATE/standard-change.md`.

Apologies for that. I will fix that on my end.

Reply:
I pushed changes fixing the standard-change @rphair.

Reply:
thanks; looks good now.